README.md

```diff
@@ -188,7 +188,7 @@ Other Style Guides
     const obj = {
       id: 5,
       name: 'San Francisco',
-      [getKey('enabled')]: true,
+      getKey('enabled'): true,
```

Comment:
this, in fact, would cause a syntax error - the original version with the brackets is indeed correct and equivalent.